boards/xtensa/esp32/esp32_defconfig

```diff
@@ -18,6 +18,8 @@ CONFIG_XTENSA_USE_CORE_CRT1=n
 CONFIG_PINMUX=y
 CONFIG_PINMUX_ESP32=y
 
+CONFIG_PINCTRL=y
```

Comment:
suggestion: default PINCTRL to y at SoC level if it's always required.

Reply:
Moved to SoC level. Thanks for the suggestion.